Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel+FaceDetectionResultHandler.swift

```diff
@@ -104,16 +104,30 @@ extension FaceLivenessDetectionViewModel: FaceDetectionResultHandler {
 
     func handleInstruction(
         _ instruction: Instructor.Instruction,
-        colorSequences: [FaceLivenessSession.DisplayColor]
+        colorSequences: [FaceLivenessSession.DisplayColor]?
     ) {
         DispatchQueue.main.async {
             switch instruction {
             case .match:
                 self.livenessState.faceMatched()
                 self.faceMatchedTimestamp = Date().timestampMilliseconds
-                DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
-                    self.livenessViewControllerDelegate?.displayFreshness(colorSequences: colorSequences)
+                
+                // next step after face match
+                switch self.challenge?.type {
+                case .faceMovementAndLightChallenge:
+                    if let colorSequences = colorSequences {
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
+                            self.livenessViewControllerDelegate?.displayFreshness(colorSequences: colorSequences)
+                        }
+                    }
+                case .faceMovementChallenge:
+                    DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
```

Comment:
the 1 second delay was an intended UX design for the end-user to hold their face during the color challenge.  Can you double check or confirm if this is necessary anymore? If not, the second delay can be removed.

Reply:
Since there is no light challenge, we should not display the "Hold Still" instruction text either.

Reply:
i can confirm with UX in a follow up PR. Made changes for `Hold still`